internal/workload/manager.go

```diff
@@ -266,86 +259,6 @@ func (w *WorkloadManager) toPodYaml(pod *v1.Pod) ([]byte, error) {
 	return podYaml, nil
 }
 
-func (w *WorkloadManager) ensureWorkloadsFromManifestsAreRunning() error {
```

Comment:
With workload monitoring gone, please remove any notion of monitoring in tests; like here:https://github.com/jakub-dzon/k4e-device-worker/pull/54/files#diff-b99535cc9b7dcd200727082cda007cd8122ac6760506c4c0f825a9a1efc72b42R56